GoogleForCreators/web-stories-wp
# Remove 'upload' button while a search is active

## Summary
Add a state control to display the search result during the searching event on media library.

## Relevant Technical Choices

Add a conditional rendering on the search result display.

## To-do

N/A

## User-facing changes

See screenshots at the comments

## Testing Instructions

Case 1:
User enter a keyword on the search bar -> hit `Enter`, it will have the followings:
- The `Upload` button should gone
- The `Upload` button should be replace by text `n result(s) found`

Case 2:
When user is using the filter based on Case 1, if there is no result found
- The filter dropdown should still stay there, so that user could switch to other filters
- The text `No media found` should show under the filter

Case 3:
When user clean up the search keyword
- The `Upload` button should come back
- The `n result(s) found` text should gone

---



Fixes #2235
